backend/capellacollab/navbar/routes.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import fastapi
+from sqlalchemy import orm
+
+from capellacollab.core import database
+from capellacollab.settings.configuration import core as config_core
+from capellacollab.settings.configuration import (
+    models as settings_config_models,
+)
+from capellacollab.settings.configuration.models import NavbarConfiguration
+
+router = fastapi.APIRouter()
+
+
+@router.get(
+    "/navbar",
+    response_model=NavbarConfiguration,
+)
+def get_navbar(db: orm.Session = fastapi.Depends(database.get_db)):
```

Comment:
It's not a complex route and there is not much that can go wrong, but a small test calling the API endpoint would be great. Just to make sure that the validation itself works properly and we don't break the route in future refactorings.